<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="NEWNODE64" ignored="0" name="Test" tests="9" failures="2" timestamp="27 Jul 2018 06:52:25 GMT" time="0.044" errors="0">
  <testcase name="Test1" time="0.008" classname="com.qa.retryLoginInSelenium.MyTests">
    <skipped/>
  </testcase> <!-- Test1 -->
  <testcase name="Test1" time="0.001" classname="com.qa.retryLoginInSelenium.MyTests">
    <skipped/>
  </testcase> <!-- Test1 -->
  <testcase name="Test1" time="0.001" classname="com.qa.retryLoginInSelenium.MyTests">
    <skipped/>
  </testcase> <!-- Test1 -->
  <testcase name="Test1" time="0.0" classname="com.qa.retryLoginInSelenium.MyTests">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.qa.retryLoginInSelenium.MyTests.Test1(MyTests.java:14)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
... Removed 22 stack frames]]>
    </failure>
  </testcase> <!-- Test1 -->
  <testcase name="Test2" time="0.001" classname="com.qa.retryLoginInSelenium.MyTests">
    <skipped/>
  </testcase> <!-- Test2 -->
  <testcase name="Test2" time="0.004" classname="com.qa.retryLoginInSelenium.MyTests">
    <skipped/>
  </testcase> <!-- Test2 -->
  <testcase name="Test2" time="0.0" classname="com.qa.retryLoginInSelenium.MyTests">
    <skipped/>
  </testcase> <!-- Test2 -->
  <testcase name="Test2" time="0.001" classname="com.qa.retryLoginInSelenium.MyTests">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.qa.retryLoginInSelenium.MyTests.Test2(MyTests.java:20)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
... Removed 22 stack frames]]>
    </failure>
  </testcase> <!-- Test2 -->
  <testcase name="Test3" time="0.0" classname="com.qa.retryLoginInSelenium.MyTests"/>
</testsuite> <!-- Test -->
